plugins/outputs/azure_data_explorer/azure_data_explorer.go

```diff
@@ -230,7 +230,7 @@ func (adx *AzureDataExplorer) Init() error {
 		return errors.New("Metrics grouping type is not valid")
 	}
 
-	serializer, err := json.NewSerializer(time.Second)
+	serializer, err := json.NewSerializer(time.Second, "") // FIXME: get the json.TimestampFormat from the config file
```

Comment:
Just noticed this, seems like this should be resolved before we merge this PR so that it can be used in this plugin? What plugin are you planning on using `json_timestamp_layout`?

Reply:
The `http` output plugin is the one we use the `json_timestamp_format` (or "layout") with. But, as the *azure* output plugin seems to make use of the Json serializer too, I had to add the additional (empty) parameter.

Reply:
Is there anything I should do here? Or is this explanation sufficient?

Reply:
ping